warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml

```diff
@@ -46,15 +46,21 @@ models:
       - name: agency_name
         description: '{{ doc("gtfs_agency__agency_name") }}'
         tests:
-        - not_null
+          - dbt_utils.not_null_proportion:
+              at_least: 0.99
+              severity: warn
       - name: agency_url
         description: '{{ doc("gtfs_agency__agency_url") }}'
         tests:
-        - not_null
+          - dbt_utils.not_null_proportion:
+              at_least: 0.99
+              severity: warn
       - name: agency_timezone
         description: '{{ doc("gtfs_agency__agency_timezone") }}'
         tests:
-        - not_null
+        - not_null:
```

Comment:
Ok sorry this is an incredibly annoying suggestion, but I think it would be worth it to add a YAML anchor to name this like `&not_null_warn` and then use that below, to limit how much more verbose this makes the YAML. 

Could do the same for the .99 threshold: `&not_null_warn_99_threshold` or something